app/screens/post_options/post_options.test.js

```diff
@@ -52,37 +61,44 @@ describe('PostOptions', () => {
         theme: Preferences.THEMES.default,
     };
 
-    test('should match snapshot, showing all possible options', () => {
-        const wrapper = shallow(
-            <PostOptions {...baseProps}/>,
-            {context: {intl: {formatMessage: jest.fn((format) => format.defaultMessage)}}},
+    function getWrapper(props={}) {
+        return shallow(
+            <PostOptions
+                {...baseProps}
+                {...props}
+            />,
+            {context: {intl: {formatMessage: ({defaultMessage}) => defaultMessage}}}
         );
+    }
+
+    test('should match snapshot, showing all possible options', () => {
+        const wrapper = getWrapper();
 
         expect(wrapper.getElement()).toMatchSnapshot();
     });
 
     test('should match snapshot, showing Delete option only for system message to user who has permission to delete', () => {
-        const wrapper = shallow(
-            <PostOptions
-                {...baseProps}
-                isSystemMessage={true}
-            />,
-            {context: {intl: {formatMessage: jest.fn((format) => format.defaultMessage)}}},
-        );
+        const wrapper = getWrapper({isSystemMessage: true});
 
         expect(wrapper.getElement()).toMatchSnapshot();
     });
 
     test('should match snapshot, no option for system message to user who doesn\'t have the permission to delete', () => {
-        const wrapper = shallow(
-            <PostOptions
-                {...baseProps}
-                isSystemMessage={true}
-                canDelete={false}
-            />,
-            {context: {intl: {formatMessage: jest.fn((format) => format.defaultMessage)}}},
-        );
+        const wrapper = getWrapper({isSystemMessage: true, canDelete: false});
 
         expect(wrapper.getElement()).toMatchSnapshot();
     });
+
+    test('should load thread', () => {
+        const wrapper = getWrapper();
+
+        wrapper.findWhere((node) => node.key() == 'reply').simulate('press');
```

Comment:
this is the reason why is failing.

```suggestion
        wrapper.findWhere((node) => node.key() === 'reply').simulate('press');
```

Reply:
Sorry, I ran `npm test` instead of `make test` before committing so I missed the linting errors!